app/views/users/edit.html.erb

```diff
@@ -1,4 +1,11 @@
 <h1>Edit your profile</h1>
+
+<% flash.each do |name, msg| %>
```

Comment:
might be useful to move this to the primary view if we end up reusing for tips and favorites